staging/src/k8s.io/apimachinery/pkg/runtime/types.go

```diff
@@ -99,6 +104,16 @@ type RawExtension struct {
 	Object Object `json:"-"`
 }
 
+func (_ RawExtension) OpenAPIDefinition() openapi.OpenAPIDefinition {
+	return openapi.OpenAPIDefinition{
+		Schema: spec.Schema{
+			SchemaProps: spec.SchemaProps{
+				Description: "Any value (any object, array or primitive)",
+			},
```

Comment:
I would add a `"type": "object"` here to be more explicit.

Reply:
Yes, I've already changed that (but have yet to update the PR). My initial understanding was that `RawExtension` was used for any type, not just for objects.